extras/lede_generic/gl_b1300/info/scan_nl80211.txt

```diff
@@ -0,0 +1,346 @@
+          }
+        },
+        {
+          "bssid": "1c:49:7b:d6:7c:18",
+          "ssid": "7665656b7368696368657272",
+          "chan": 56,
+          "rssi": -10,
+          "extras": {
+            "ht_width": "any",
+            "vht_width": "80 MHz",
+            "encryption": "Group : CCMP, Pairwise: CCMP, Auth: PSK"
+          }
+        },
+        {
+          "bssid": "c4:e9:0a:dd:c9:c0",
+          "ssid": "6b736c766e657735",
+          "chan": 149,
+          "rssi": -90,
+          "extras": {
+            "ht_width": "any",
+            "vht_width": "80 MHz",
+            "encryption": "Group : CCMP, Pairwise: CCMP, Auth: PSK"
+          }
+        },
+        {
+          "bssid": "78:17:35:34:a2:0d",
+          "ssid": "53726176616e5f35",
+          "chan": 149,
+          "rssi": -89,
+          "extras": {
+            "ht_width": "any",
+            "vht_width": "80 MHz",
+            "encryption": "Group : TKIP, Pairwise: TKIP CCMP, Auth: PSK"
+          }
+        },
+        {
+          "bssid": "60:38:e0:11:a1:c3",
+          "ssid": "4c4e352e",
+          "chan": 153,
+          "rssi": -79,
+          "extras": {
+            "ht_width": "any",
+            "vht_width": "80 MHz",
+            "encryption": "Group : TKIP, Pairwise: CCMP, Auth: PSK"
+          }
+        },
+        {
+          "bssid": "1c:49:7b:d6:7c:1b",
+          "ssid": "",
+          "chan": 157,
+          "rssi": -7,
+          "extras": {
+            "ht_width": "any",
+            "vht_width": "80 MHz",
+            "encryption": "Group : CCMP, Pairwise: CCMP, Auth: PSK"
+          }
+        },
+        {
+          "bssid": "80:02:9c:34:bb:a8",
+          "ssid": "",
+          "chan": 157,
+          "rssi": -76,
+          "extras": {
+            "ht_width": "any",
+            "vht_width": "80 MHz",
+            "encryption": "Group : CCMP, Pairwise: CCMP, Auth: PSK"
+          }
+        },
+        {
+          "bssid": "80:02:9c:34:bb:a9",
+          "ssid": "7665656b7368696368657272",
+          "chan": 157,
+          "rssi": -76,
+          "extras": {
+            "ht_width": "any",
+            "vht_width": "80 MHz",
+            "encryption": "Group : CCMP, Pairwise: CCMP, Auth: PSK"
+          }
+        },
+        {
+          "bssid": "80:02:9c:34:bb:aa",
+          "ssid": "47727970686f6e4775657374376331",
+          "chan": 157,
+          "rssi": -76,
+          "extras": {
+            "ht_width": "any",
+            "vht_width": "80 MHz",
+            "encryption": "Group : CCMP, Pairwise: CCMP, Auth: PSK"
+          }
+        }
+      ]
+    },
+    {
+      "name": "phy0",
+      "scanlist": [
+        {
+          "bssid": "78:17:35:bf:ea:49",
+          "ssid": "56656e6b6174614169",
+          "chan": 1,
+          "rssi": -28,
+          "extras": {
+            "ht_width": "any",
+            "vht_width": "20 or 40 MHz",
+            "encryption": "Group : TKIP, Pairwise: TKIP CCMP, Auth: PSK"
```

Comment:
It looks like you forgot to change the example... (it should be `...Pairwise:TKIP/CCMP ...`)
BTW, please have the WEP and Open SSIDs in the example as well.

Reply:
Or is it a stale file? It looks like scan.txt is up-to-date, just needs "WEP" and "open" entries.